cristin-person/src/main/java/no/unit/nva/cristin/person/institution/update/PersonInstInfoPatchSerializer.java

```diff
@@ -5,10 +5,14 @@
 import com.fasterxml.jackson.databind.SerializerProvider;
 import com.fasterxml.jackson.databind.ser.std.StdSerializer;
 import java.io.IOException;
+import java.io.Serial;
 import no.unit.nva.cristin.person.model.nva.PersonInstInfoPatch;
 
 public class PersonInstInfoPatchSerializer extends StdSerializer<PersonInstInfoPatch> {
 
+    @Serial
+    private static final long serialVersionUID = -3456674536341L;
```

Comment:
Can we also exclude this from pmd or should we keep it? what does it do? do we need it? Baeldung states this:
````
If serialVersionUID is not provided in a Serializable class, the JVM will generate one automatically. However, it is good practice to provide the serialVersionUID value and update it after changes to the class so that we can have control over the serialization/deserialization process. We'll take a closer look at it in a later section.
````